common/transport/amqp/devdoc/receiver_link_requirements.md

```diff
@@ -42,33 +42,33 @@ receiverLink.on('message', (msg) => {
 
 ### attach(callback: (err?: Error) => void): void
 
-**SRS_NODE_AMQP_RECEIVER_LINK_16_004: [** The `attach` method shall use the stored instance of the `amqp10.AmqpClient` object to attach a new link object with the `linkAddress` and `linkOptions` provided when creating the `ReceiverLink` instance. **]**
+**SRS_NODE_AMQP_RECEIVER_LINK_16_004: [** The `attach` method shall use the stored instance of the `rhea` session object to attach a new link object with the combined `linkAddress` and `linkOptions` provided when creating the `ReceiverLink` instance. **]**
```

Comment:
>session [](start = 106, length = 7)

same question about that session object. the requirement documents should reflect how we use the whole rhea API